stravalib/exc.py

```diff
@@ -113,3 +113,13 @@ def warn_param_unofficial(param_name: str):
         FutureWarning,
         stacklevel=3
     )
+
+
+def warn_units_deprecated():
```

Comment:
A warning to discourage users from using the legacy `units` types and attributes so we can eventually get rid of the monstrosities in the `unithelper` module.

Reply:
😆 thanks for the laugh. cya monstrosities!